tck/features/LabelsAcceptance.feature

```diff
@@ -0,0 +1,264 @@
+      | +labels | 1 |
+
+  Scenario: Ignore space before colon
+    And having executed:
+      """
+      CREATE ()
+      """
+    When executing query:
+      """
+      MATCH (n)
+      SET n :Foo
+      RETURN labels(n)
+      """
+    Then the result should be:
+      | labels(n) |
+      | ['Foo']   |
+    And the side effects should be:
+      | +labels | 1 |
+
+  Scenario: Adding multiple labels
+    And having executed:
+      """
+      CREATE ()
+      """
+    When executing query:
+      """
+      MATCH (n)
+      SET n:Foo:Bar
+      RETURN labels(n)
+      """
+    Then the result should be:
+      | labels(n)      |
+      | ['Foo', 'Bar'] |
+    And the side effects should be:
+      | +labels | 2 |
+
+  Scenario: Ignoring intermediate whitespace 1
+    And having executed:
+      """
+      CREATE ()
+      """
+    When executing query:
+      """
+      MATCH (n)
+      SET n :Foo :Bar
+      RETURN labels(n)
+      """
+    Then the result should be:
+      | labels(n)      |
+      | ['Foo', 'Bar'] |
+    And the side effects should be:
+      | +labels | 2 |
+
+  Scenario: Ignoring intermediate whitespace 2
+    And having executed:
+      """
+      CREATE ()
+      """
+    When executing query:
+      """
+      MATCH (n)
+      SET n :Foo:Bar
+      RETURN labels(n)
+      """
+    Then the result should be:
+      | labels(n)      |
+      | ['Foo', 'Bar'] |
+    And the side effects should be:
+      | +labels | 2 |
+
+  Scenario: Creating node without label
+    When executing query:
+      """
+      CREATE (node)
+      RETURN labels(node)
+      """
+    Then the result should be:
+      | labels(node) |
+      | []           |
+    And the side effects should be:
+      | +nodes | 1 |
+
+  Scenario: Creating node with two labels
+    When executing query:
+      """
+      CREATE (node:Foo:Bar {name: 'Mattias'})
+      RETURN labels(node)
+      """
+    Then the result should be:
+      | labels(node)   |
+      | ['Foo', 'Bar'] |
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +labels     | 2 |
+      | +properties | 1 |
+
+  Scenario: Ignore space when creating node with labels
+    When executing query:
+      """
+      CREATE (node :Foo:Bar)
+      RETURN labels(node)
+      """
+    Then the result should be:
+      | labels(node)   |
+      | ['Foo', 'Bar'] |
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 2 |
+
+  Scenario: Create node with label in pattern
+    When executing query:
+      """
+      CREATE (n:Person)-[:OWNS]->(x:Dog)
+      RETURN labels(n)
+      """
+    Then the result should be:
+      | labels(n)  |
+      | ['Person'] |
+    And the side effects should be:
+      | +nodes         | 2 |
+      | +relationships | 1 |
+      | +labels        | 2 |
+
+  Scenario: Fail when adding new label predicate on already bound node 1
+    When executing query:
+      """
+      CREATE (n: Foo)-[:T1]->(b), (n: Bar)-[:T2]->(c)
```

Comment:
`(n: Bar)-[:T2]->(c)` ought to be on a new line, and there are spaces between the colons and the labels (which this test is not testing for)